doc/classes/EditorSettings.xml

```diff
@@ -734,6 +841,12 @@
 			Depending on the platform and used renderer, the engine will fall back to [b]Enabled[/b] if the desired mode is not supported.
 			[b]Note:[/b] V-Sync modes other than [b]Enabled[/b] are only supported in the Forward+ and Mobile rendering methods, not Compatibility.
 		</member>
+		<member name="interface/editors/derive_script_globals_by_name" type="bool" setter="" getter="">
+			If [code]true[/code], when extending a custom script and it has a global name, the Inherits field of script creation dialog will be filled with the global name instead of script path.
+		</member>
+		<member name="interface/editors/show_scene_tree_root_selection" type="bool" setter="" getter="">
+			If [code]true[/code], when an empty scene is opened, Scene dock will display buttons for quickly creating a root node.
```

Comment:
I know this is true, but in practice most users will see this when _creating_ a new scene.
```suggestion
			If [code]true[/code], the Scene dock will display buttons to quickly add a root node to a newly created scene.
```